apps/language_models/scripts/vicuna.py

```diff
@@ -103,14 +103,7 @@
 )
 
 
-def brevitas〇matmul_rhs_group_quant〡shape(
-    lhs: List[int],
-    rhs: List[int],
-    rhs_scale: List[int],
-    rhs_zero_point: List[int],
-    rhs_bit_width: int,
-    rhs_group_size: int,
-) -> List[int]:
+def brevitas〇matmul_rhs_group_quant〡shape(lhs: List[int], rhs: List[int], rhs_scale: List[int], rhs_zero_point: List[int], rhs_bit_width: int, rhs_group_size: int) -> List[int]:
```

Comment:
@stellaraccident  do you know how this changes the function signature.

Reply:
And @makslevental  any thoughts

Reply:
I think this is because of the way these functions are verified in Torch-MLIR. They match the functions' first line for verifying the signature (https://github.com/llvm/torch-mlir/blob/main/python/torch_mlir/dialects/torch/importer/jit_ir/build_tools/library_generator.py#L177), and after formatting the function definition goes to multiple lines and as a result, it would fail the verification.